charts/incubator/jellystat/templates/_secrets.tpl

```diff
@@ -0,0 +1,14 @@
+{{/* Define the secrets */}}
+{{- define "jellystat.secrets" -}}
+{{- $secretName := (printf "%s-jellystat-secrets" (include "tc.v1.common.lib.chart.names.fullname" $)) }}
+{{- $jellystatprevious := lookup "v1" "Secret" .Release.Namespace $secretName }}
+enabled: true
+data:
+  {{- if $jellystatprevious }}
+  JWT_SECRET: {{ index $jellystatprevious.data "JWT_SECRET" | b64dec }}
+  {{- else }}
+  {{- $JWT_SECRET := randAlphaNum 64 }}
+  JWT_SECRET: {{ $JWT_SECRET }}
+  {{- end }}
+
```

Comment:
Can we convert this like this please?
https://github.com/truecharts/charts/blob/10b88d86a8a7b2b98195fc1dafbe745f64d043dd/charts/stable/vikunja/templates/_secrets.tpl#L6-L9